frontend/js/modules/search.js

```diff
@@ -0,0 +1,179 @@
+module.exports = (function () {
+    'use strict';
+    // Global dependencies
+    var m  = require('mithril');
+    var ld = require('lodash');
+  
+    // Local dependencies
+    var conf      = require('../configuration.js');
```

Comment:
I know mypads uses this indentation scheme, but I suggest against it.  Better to just have one space after the variable name and then the assignment already.  (My main issue with this, is it will require multiple line edits in case a longer named var is added.